client/src/components/RecipeForm/index.jsx

```diff
@@ -39,6 +39,14 @@ const RecipeForm = ({ title, recipe, setShowModal }) => {
       setTags((prev) => [...prev, trimmedInput]);
       setInput('');
     }
+
+    if (e.key === 'Backspace' && !input.length && tags.length) {
```

Comment:
Suggestion: Delete whole tag in one backspace
Suggestion#2: Maybe not have this functionality at all